docs/docusaurus/docs/deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_redshift.md

```diff
@@ -68,18 +63,10 @@ import IdentifyDataContextExpectationsStore from '@site/docs/guides/setup/config
 
 import AddS3ExpectationsStoreConfiguration from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_update_your_configuration_file_to_include_a_new_store_for_expectations_on_s.mdx'
 
-<!-- 1.4.3 Verify that the new Amazon S3 Expectations Store has been added successfully -->
-
-import VerifyS3ExpectationsStoreExists from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_confirm_that_the_new_expectations_store_has_been_added_by_running_great_expectations_store_list.mdx'
```

Comment:
if these files aren't references elsewhere, can they be deleted?

Reply:
I'll grep around

Reply:
All of the various md and mdx files that are no longer referenced in this PR are being used by at least 1 other file.